Source/WebKit/Shared/Extensions/WebExtensionContextParameters.h

```diff
@@ -45,7 +45,7 @@ struct WebExtensionContextParameters {
 
     HashMap<String, WallTime> grantedPermissions;
 
-    Ref<API::Data> localizationJSON;
+    RefPtr<API::Data> localizationJSON;
```

Comment:
Why can't it stay Ref to signal it is never missing / null?

Reply:
It can be null, in `serializeLocalization` if the localization class isn't created or there's no localization JSON saved, the function returns nullptr, and I beleive using something like `{ }` didn't work initially.